_data/projects/react-test.yml

```diff
@@ -0,0 +1,16 @@
+name: React Test
+desc: An expressive testing library for React
+site: http://react-test.dev/
+tags:
+  - reactjs
+  - test
+  - testing
+  - jquery
+  - expressive
+  - act
+  - render
+  - jest
+  - runner
+upforgrabs:
+  name: good first issue
+  link: https://github.com/franciscop/react-test/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
@franciscop Please change this to https://github.com/franciscop/react-test/labels/good%20first%20issue